modules/aws_asg/main.tf

```diff
@@ -332,9 +332,23 @@ resource "aws_autoscaling_policy" "workers_add_capacity" {
   estimated_instance_warmup = "${var.worker_asg_scale_out_cooldown}"
   metric_aggregation_type   = "Maximum"
 
+  # Headroom is just below THRESHOLD, scale out normally
   step_adjustment {
     scaling_adjustment          = "${var.worker_asg_scale_out_qty}"
-    metric_interval_lower_bound = 1.0
+    metric_interval_lower_bound = "${var.worker_asg_scale_out_threshold * -1/3 -1}"
```

Comment:
I don't understand the intent here 😕 

When `worker_asg_scale_out_threshold` has a value of `80`, as is the case for one of the ASGs in production, then `${var.worker_asg_scale_out_threshold * -1/3 -1}` gives me `-27.6`.  Does terraform get a different result?  What _should_ it be?

Reply:
Something like "when we're below the threshold, create n instances. When we're at 50% of the threshold, create n * 2 instances."
I pushed another commit with less weird torturous math.